Source/WebCore/css/CSSValue.h

```diff
@@ -54,92 +56,98 @@ DECLARE_ALLOCATOR_WITH_HEAP_IDENTIFIER(CSSValue);
 class CSSValue {
```

Comment:
It would be good to add a comment giving an overview on how the data-in-the-pointer scheme here works.

Reply:
Yes, makes sense, I will add a comment.

Reply:
I added one.